### PR TITLE
TPB

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
I fear this needs to be updadated, because some of them may be down now.
Could you explain the difference between all the files? You should have a look at our existing ThePirateBay.xml

Please add some coments showing the relation between all the files, if there are any.

**Comment by J0WI** [comment]
Note that you also need to mention these rules in ThePirateBay.xml

**Comment by galeksandrp** [comment]
Added.

File: src/chrome/content/rules/ThePirateBay.xml [hunk 1]
```diff
@@ -14,9 +15,20 @@
 	<target host="kanyebay.com" />
 	<target host="www.kanyebay.com" />
 
-	<!-- Should probably split the proxy stuff out to a separate module and cover all the proxies listed at thepiratebay-proxylist.org -->
+	<!-- Other The Pirate Bay rulesets:
```
Comment (J0WI) [inline src/chrome/content/rules/ThePirateBay.xml]: Please move this comment block on the top.

**Comment by J0WI** [comment]
Thanks for the additions! There is a similar PR #4474, could you therefore adding the changes from there:
Removing Old_Pirate_Bay.org.xml
Adding tpb.run www.tpb.run.

Please remove the unrelated change in translations.

**Comment by galeksandrp** [comment]
Added.

**Comment by J0WI** [comment]
Thanks for all the additions!

**Pull request merged**